vpr/src/device/rr_graph_builder.h

```diff
@@ -273,6 +273,10 @@ class RRGraphBuilder {
     inline void init_fan_in() {
         node_storage_.init_fan_in();
     }
+    /** @brief Return number of nodes. This function is inlined for runtime optimization. */
+    inline size_t size() const {
```

Comment:
We can use the API name ``num_nodes()`` here, we can also change the API name in ``RRGraphView``